include/brickengine/input.hpp

```diff
@@ -398,6 +398,19 @@ class BrickInput {
         sdl_mapping.insert({InputKeyCode::Ekey_enter, SDLK_KP_ENTER});
         sdl_mapping.insert({InputKeyCode::EKey_backspace, SDLK_BACKSPACE});
 
+        sdl_mapping.insert({InputKeyCode::EKey_f1, SDLK_F1});
```

Comment:
To follow the previous 'inserted' keys, I would add the comment // F keys above this section of inserted F keys.

Reply:
I agree